### DISCUSSION TITLE
What the heck is "Enter a visit purpose"?

### DISCUSSION THREAD

**Discussion by acoolov** [YetiForceCompany/YetiForceCRM#15304] [How to] [answered]
Hello,

Just installed a fresh 6.2 instance and once I log in, it gives me a message "Enter a visit purpose"

What the heck is this annoying message and how do I disable it? It is so annoying, why does Yetiforce keep adding some of these super annoying functionality??

**Comment by rskrzypczak** [comment, marked as answer]
Before adding a post use the search engine.
https://github.com/YetiForceCompany/YetiForceCRM/discussions?discussions_q=Enter+a+visit+purpose
Here's the answer: https://github.com/YetiForceCompany/YetiForceCRM/discussions/15295

**Comment by acoolov** [comment]
Thanks, I appreciate it. When I try to change it through the web gui and go to Settings > Admin Access it loads a blank page and gives a message saying "Error, please inform the administrator". What could this be? both Admin Access and Visit Purpose tabs are blank.

**Reply by mariuszkrzaczkowski** [reply]
create an issue with full details

**Reply by rskrzypczak** [reply]
Install the latest service pack (https://github.com/YetiForceCompany/YetiForceCRM/releases/tag/6.2.0)
